docs/data/galexie/admin_guide/prerequisites.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Prerequisites
+sidebar_position: 10
+---
+
+# Prerequisites
+
+### 1. Google Cloud Platform (GCP) Account
+
+Galexie exports Stellar ledger metadata to Google Cloud Storage (GCS), so you need a GCP account with:
+
+- Permissions to create a new GCS bucket, or
+- Access to an existing bucket with read/write permissions.
+
+### 2. Docker (Recommended)
+
+> **_NOTE:_** While it is possible to natively install Galexie (without Docker), this requires manual dependency management and is recommended only for advanced users.]
+
+Galexie is available as a Docker image, which simplifies installation and setup. Ensure you have Docker Engine installed on your system ([Docker installation guide](https://docs.docker.com/engine/install/)).
+
+## Hardware Requirements
+
+The minimum hardware requirements for running Galexie are:
```

Comment:
Pubnet Galexie runs with GCP `e2-standard-2` (2 vCPU, 8 GB mem) and the default 100 GB boot disk for storage
Testnet Galexie runs with GCP `e2-small` (0.5-2 vCPU, 2 GB mem) and the default 100 GB boot disk for storage

> Also info on estimated time for exporting full history and guidance on scaling for faster exports

I don't think we should link to https://github.com/stellar/go/pull/5407 because it is too hacky. We should come up with a real backfill solution/recommendation to add to the public stellar docs.

What do you think about using your Horizon backfill estimates for exporting full history?
Or we could pull in some of the info from the Summary section of my [Galexie backfill notes](https://docs.google.com/document/d/1xNQA77g3QqJlerFckDKWHtHk2WynG08MgQh29wLebUo/edit?tab=t.0#heading=h.wh5j95naq7tx)